corebehrt/configs/ozempic/criteria_definitions.yaml

```
@@ -1,192 +1,257 @@
-delays:
-  days: 14 # we match diagnoses and register diagnoses up to index_date + delay_days/ assume it is not caused by the trial drug
-  code_groups:
-    - "D/" # diagnoses
-    - "RD/" # register diagnoses
+# provide expressions for inclusion and exclusion
+inclusion: type2_diabetes & (older_than_50_and_clinical_evidence_of_cvd | older_than_60_and_subclinical_evidence_of_cvd) & hba1c_level & insulin
+exclusion: max_age_60 & criteria_3 & criteria_4 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)